src/subprocess_launcher.rs

```diff
@@ -1,36 +1,36 @@
 #[cfg(any(feature = "v2_40", feature = "dox"))]
 use SubprocessLauncher;
-#[cfg(any(all(feature = "v2_40", not(windows)), feature = "dox"))]
+#[cfg(any(all(feature = "v2_40", not(unix)), feature = "dox"))]
```

Comment:
This is wrong change, as windows ~= not(unix)

Reply:
Oups indeed!